pkgs/development/python-modules/loompy/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  fetchPypi,
+  python3Packages,
+}:
+let
+  finalAttrs = {
+    pname = "loompy";
+    version = "3.0.7";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit (finalAttrs) pname version;
+      hash = "sha256-tc33tUc0xr7ToYHRGUevcK8sbg3K3AL9Docd8jL6qPQ=";
+    };
+
+    build-system = with python3Packages; [ setuptools ];
+
+    dependencies = with python3Packages; [
+      h5py
+      numpy
+      scipy
+      numba
+      click
+      numpy-groupies
+    ];
+
```

Comment:
Could you add `pythonImportsCheck`?

> pythonImportsCheck is set. This is still a good smoke test even if pytestCheckHook is set.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines